blazor/datagrid/cell-edit-types.md

```diff
@@ -32,10 +32,10 @@ component for TimeOnly data type.
 
 ## Customizing the default editor controls
 
-You can customize the behavior of the editor component through the [EditorSettings](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html) property of the [GridColumn](https://help.syncfusion.com/cr/aspnetcore-blazor/Syncfusion.Blazor.Grids.GridColumn.html) component.
+You can customize the behavior of the editor component through the [EditorSettings](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html) property of the [GridColumn](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html) component.
```

Comment:
Check all the CR links and update. The below screenshot isn't the correct link
<img width="575" alt="image" src="https://github.com/syncfusion-content/blazor-docs/assets/95272306/cc983383-ced2-4ec6-b7b4-9bf766ce9f02">

Reply:
Updated